txstratum/manager.py

```diff
@@ -237,7 +237,7 @@ def enqueue_tx_job(self, job: MinerTxJob) -> None:
         """Enqueue a tx job to be mined."""
         assert job not in self.tx_queue
         job.status = JobStatus.ENQUEUED
-        job.expected_queue_time = sum(x.expected_mining_time for x in self.tx_queue)
+        job.expected_queue_time = sum(x.expected_mining_time for x in self.tx_queue if x.expected_mining_time > 0)
```

Comment:
Maybe we could add the same description you used in the PR here as a comment. What do you think?

Reply:
Done. And I also added a new unittest. Thanks!